### PR TITLE
July 26 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | c08105f |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-490.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20240726_amplify.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+* Go to [https://repository.axway.com/catalog?q=agents](https://repository.axway.com/catalog?q=agents).
+
+### Axway Agent SDK
+
+The following enhancements and bug fixes are available in this release.
+
+#### Enhancements for Agent SDK
+
+* **External OAuth Identity Provider support** - The Agent SDK has support for external OAuth Identity Provider(s) for the on-premise and SaaS Azure Agents.
+* **Azure EventHub support** - The Agend SDK has support for the Azure EventHub configuration.
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/20240726_amplify.md:58]: Neither of these make sense, I do not think referencing any specific dataplane will ever make sense for the SDK. The SDK may be enhanced for common functionality between dataplanes but never for a specific one.

These should be:

- "Enhanced support for External IdP configuration and usage" 
- "Async API spec generation"

File: content/en/docs/amplify_relnotes/20240726_amplify.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+##### Enhancements for Azure Gateway Agent
+
+There are no enhancements in this release.
+
+##### Bug fixes for Azure Gateway Agent
+
+| Case ID     | Internal ID  | Description                                       |
+|-------------|--------------|---------------------------------------------------|
+| 01617664    | APIGOV-28417 | **Issue**: A panic error can occur if the traceability agent (v1.1.84) is set offline mode. <br />**Resolution**: A fix was made to prevent a null pointer error when in offline mode. |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/20240726_amplify.md:84]: this is v7 not Azure

File: content/en/docs/amplify_relnotes/20240726_amplify.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+Agent versions and Gateway compatibility:
+
+| API Gateway version                        | On-premise Agent version |
+|--------------------------------------------|--------------------------|
+| Axway API Management 7.6.2 SPx and 7.7 SPx | DA=v1.1.86 / TA=v1.1.86  |
+| AWS Gateway using SDK 2.0                  | DA=v1.1.71 / TA-v1.1.74  |
+| Azure latest release                       | DA=v1.1.75 / TA=v1.1.81  |
+| Istio 1.9.5                                | DA=v1.0.42 / TA=v2.0.87  |
+| Apigee Edge                                | 0.1.44                   |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/20240726_amplify.md:23]: considering the message I referenced to you and Juan this morning, maybe we should add a note about the Google adjusting the Apigee Edge API as they decommission it?

File: content/en/docs/amplify_relnotes/20240726_amplify.md (hunk 1 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240726_amplify.md:58]: edits made

File: content/en/docs/amplify_relnotes/20240726_amplify.md (hunk 2 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240726_amplify.md:84]: edits made

File: content/en/docs/amplify_relnotes/20240726_amplify.md (hunk 3 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240726_amplify.md:23]: I added an Apigee Edge limitation statement

File: content/en/docs/amplify_relnotes/20240726_amplify.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+* Go to [https://repository.axway.com/catalog?q=agents](https://repository.axway.com/catalog?q=agents).
+
+### Axway Agent SDK
+
+The following enhancements and bug fixes are available in this release.
+
+#### Enhancements for Agent SDK
+
+* **Enhanced Identity Provider support** - The Agent SDK has enhanced the support for external Identity Provider configuration and its usage.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240726_amplify.md:57]: @robertengaxway, can we say something along this line: The Agent SDK has been extended to add support for external Identity providers for Oauth.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

File: content/en/docs/amplify_relnotes/20240726_amplify.md (hunk 4 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240726_amplify.md]: reworded to include "extended to add support .."

**Pull request merged**